tests/coverage/antlr_parser/TestFileLexer.interp

```diff
@@ -0,0 +1,330 @@
+token literal names:
```

Comment:
I'm not sure how others feel but seeing file names that are Test* makes me think they are test files as opposed to files that are responsible parsing a test file format. Maybe some other naming pattern could make that clearer.

Reply:
how about `FuncTestSpec` as prefix
This file will become `FuncTestSpecLexer`